vmware-event-router/internal/config/v1alpha1/processor.go

```diff
@@ -40,6 +45,14 @@ type ProcessorConfigOpenFaaS struct {
 	Auth *AuthMethod `yaml:"auth,omitempty" json:"auth,omitempty" jsonschema:"description=Authentication configuration for this section"`
 }
 
+// ProcessorConfigKnative configures the Knative event processor
+type ProcessorConfigKnative struct {
+	// Address is the connection address to the knative broker
+	Address string `yaml:"address" json:"address" jsonschema:"required,description=knative broker address,default=http://broker-ingress.knative-eventing.svc.cluster.local/default/default"`
```

Comment:
So, here if you s/Address/duckv1.Destination/ might be more flexible as well as be easier for the user.